src/app/pages/private/LifeListPage/displayLife.tsx

```diff
@@ -0,0 +1,28 @@
+import { useListLivesQuery } from '../../../api';
+import NoItemResult from '../../../components/results/NoItemResult';
+
+const DisplayLife = () => {
+    const { data, loading } = useListLivesQuery();
+
+    if (loading) {
+        // not yet loaded
+        return null;
+    }
+
+    const badam = data?.lives || [];
+    console.log(badam)
+    if (badam.length === 0) {
+        return <NoItemResult />
+    }
+
+    const DisplayData = () => {
+        return (
+            <>
+            {badam.map((life) => <p>{life.id} {life.title} {life.fullName}</p>)}
+            </>
+        )
+    }
+    return <div> <DisplayData /> </div>;
```

Comment:
- you dont have to create `DisplayData` ; can simply next things inside the <div>
- should jump a line before return ( eslint is happy about this ? )